apps/dashboard/src/components/side-navigation/constants.ts

```diff
@@ -0,0 +1,84 @@
+import {
+  RiBarChartBoxLine,
+  RiGroup2Line,
+  RiKey2Line,
+  RiPaintBrushLine,
+  RiRouteFill,
+  RiSettings4Line,
+  RiStore3Line,
+  RiUserAddLine,
+} from 'react-icons/ri';
+import { NavItemsGroup } from './types';
+import { LEGACY_ROUTES, ROUTES } from '@/utils/routes';
+
+export const navitationItems: NavItemsGroup[] = [
+  {
+    items: [
+      {
+        label: 'Workflows',
+        icon: RiRouteFill,
+        to: ROUTES.WORKFLOWS,
+      },
+      {
+        label: 'Subscribers',
+        icon: RiGroup2Line,
+        disabled: true,
+      },
+    ],
+  },
+  {
+    label: 'Monitor',
+    items: [
+      {
+        label: 'Activity Feed',
+        icon: RiBarChartBoxLine,
+        to: LEGACY_ROUTES.ACTIVITY_FEED,
+        isExternal: true,
+      },
+    ],
+  },
+  {
+    label: 'Developer',
+    items: [
+      {
+        label: 'Integration Store',
+        icon: RiStore3Line,
+        to: LEGACY_ROUTES.INTEGRATIONS,
+        isExternal: true,
+      },
+      {
+        label: 'API Keys',
+        icon: RiKey2Line,
+        to: LEGACY_ROUTES.API_KEYS,
+        isExternal: true,
+      },
+    ],
+  },
+  {
+    label: 'Application',
+    items: [
+      {
+        label: 'Branding',
+        icon: RiPaintBrushLine,
+        to: LEGACY_ROUTES.BRANDING,
+        isExternal: true,
+      },
+      {
+        label: 'Settings',
+        icon: RiSettings4Line,
+        to: LEGACY_ROUTES.SETTINGS,
+        isExternal: true,
```

Comment:
Lovely modelling. 

Since we call the constants `LegacyRoute` should we maybe call this flag isLegacy as well? Ignore this comment if `external` is used for cross origin redirections.

Reply:
it will be for the docs